styles/trusted-customer.css

```diff
@@ -0,0 +1,60 @@
+.trusted-customer__slider {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 60px;
+}
+.trusted-customer__item {
+  border: 2px solid #dddddd;
+  border-radius: 10px;
+  padding: 30px;
+  max-width: 400px;
+  width: 100%;
+  position: relative;
+}
+.trusted-customer__datauser {
+  display: flex;
+  align-items: center;
+  margin-bottom: 8px;
+}
+.trusted-customer__rating {
+  display: flex;
+  position: absolute;
+  top: 48px;
+  right: 30px;
+}
+.trusted-customer__username {
+  margin-left: 20px;
+}
+.trusted-customer__name {
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 30px;
+}
+.trusted-customer__born {
+  font-size: 14px;
+  color: #4f5665;
+}
+.trusted-customer__text {
+  font-size: 16px;
+  line-height: 30px;
+  margin-right: 10px;
+}
+.trusted-customer__button {
+  border: 2px solid #f53838;
+  border-radius: 50px;
+  width: 60px;
+  height: 60px;
+  padding: 20px;
+}
+.button--left {
+  background-color: #ffffff;
+  margin-right: 20px;
+}
+.button--right {
+  background-color: #f53838;
+}
```

Comment:
In fact I don't think that these buttons are different because one is left and the other one is right. Their direction are already represented with arrows. But their colors differ because they have different states — I assume it's normal and disabled. Presumably, the left one on the design is white (disabled) because we see the first slide and cannot swipe to the left. So the left arrow is inactive, not clickable — or disabled.

OR this difference is because of normal/hover states.

I mean, we need to give these classes more thoughtful names, like button--inactive for disabled state. And the normal state I would describe in the main class, `trusted-customer__button` or `button`.

All in all, it's up to you to decide which logic to choose. In real life you would ask a designer. Here — just think.

Reply:
++